src/poolMetas.ts

```diff
@@ -107,16 +107,28 @@ export let poolMetas: PoolMeta[] = [
     startBlock: 9993512
   },
 
-  // Kovan staging
+  // Kovan NAV Pool 1
```

Comment:
Can we change the name here as follows (and `Kovan NAV Pool 2` likewise)?

```suggestion
  // Kovan v2 Static NAV Pool 1
```

(We would then also have `Kovan v3 Revolving NAV Pool 1`)